test/kixi/integration/base.clj

```diff
@@ -31,9 +32,14 @@
        (clojure.test/do-report {:type :error :message "Exception diffing"
                                 :expected nil :actual t#}))))
 
+(defn instrument-specd-functions
+  []
+  (stest/instrument 'kixi.datastore.web-server/return-error))
+
 (defn cycle-system-fixture
   [all-tests]
   (repl/start)
+  (instrument-specd-functions)
```

Comment:
Well, from the little bit I read you have to switch these things on during tests. The spec is not ref'd at all for any runtime checking.